Makefile

```diff
@@ -290,40 +311,46 @@ ifndef HAS_CHECKMAKE
 endif
 	@checkmake Makefile
 
-.PHONY: docker-login
-docker-login: ## Log in into the Docker repository
+.PHONY: cr-login
+cr-login: ## Log in into the Docker repository
+	@echo "+ $@"
+
+.PHONY: cr-build
+cr-build: check-env ## Build the container
 	@echo "+ $@"
+	$(CR) build . -t $(DOCKER_REGISTRY):$(GITCOMMIT) --file build/Dockerfile $(CR_EXTRA_ARGS)
 
-.PHONY: docker-build
-docker-build: check-env ## Build the container
+.PHONY: cr-images
+cr-images: ## List all local containers
 	@echo "+ $@"
-	docker build . -t $(DOCKER_REGISTRY):$(GITCOMMIT) --file build/Dockerfile
+	$(CR) images $(CR_EXTRA_ARGS)
 
-.PHONY: docker-images
-docker-images: ## List all local containers
+.PHONY: cr-push
+cr-push: ## Push the container
```

Comment:
I guess you mean to say "Push the image", a container is an instance of the image.